Dockerfile.livecd-iso-image

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
+RUN mkdir /root/image
+COPY ./rhcos-46.82.202008111140-0-live.x86_64.iso /root/image/livecd.iso
```

Comment:
@mvazquezc @yevgeny-shnaidman cannot we use wget here? instead of letting the user to download it to his laptop in predefined step?

Reply:
Yes, I had that task pending. It's updated now.